### PR TITLE
[C# Samples Post 1.16 release] Refactor code samples to use OrtValue API

### PR DESCRIPTION
**Pull request by yuslepukhin**
This is a pending PR that should be merged when ORT 1.16 releases.

### CODE REVIEW

File: c_sharp/Snpe_EP/vgg16_image_classification/image_classification.csproj [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <Platforms>AnyCPU;ARM64</Platforms>
+    <Platforms>AnyCPU</Platforms>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="1.16.0-dev-20230704-1234-d540c7da0f" />
+    <PackageReference Include="Microsoft.ML.OnnxRuntime.Managed" Version="1.16.0-dev-20230704-1234-d540c7da0f" />
     <PackageReference Include="Microsoft.ML.OnnxRuntime.Snpe" Version="1.13.1" />
```
Comment (yuslepukhin) [inline c_sharp/Snpe_EP/vgg16_image_classification/image_classification.csproj:14]: ><PackageReference Include="Microsoft.ML.OnnxRuntime.Snpe" Version="1.13.1" /> [](http://example.com/codeflow?start=4&length=77)

Will need to update Snpe and remove Onnxruntime

**Comment by yuslepukhin** [comment]
All Onnxruntime packages will flip to the released ones once available. Currently dev used for testing.

File: c_sharp/Snpe_EP/vgg16_image_classification/image_classification.csproj (hunk 1 above)
Comment (HectorSVC) [reply c_sharp/Snpe_EP/vgg16_image_classification/image_classification.csproj:14]: We deprecated SNPE EP. I think we should remove this code example for SNPE EP.

---
In reply to: [1253755250](https://github.com/microsoft/onnxruntime-inference-examples/pull/270#discussion_r1253755250) [](ancestors = 1253755250)

Comment (yuslepukhin) [reply c_sharp/Snpe_EP/vgg16_image_classification/image_classification.csproj]: Removed

**Review by HectorSVC — APPROVED** [review]
:shipit:

**Review by snnn — APPROVED** [review]

**Pull request merged**